<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgNuevaMisionFilipinas</class>
 <widget class="QDialog" name="dlgNuevaMisionFilipinas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Misión nº</string>
       </property>
       <property name="buddy">
        <cstring>spMisionNumero</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spMisionNumero">
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Salida desde</string>
       </property>
       <property name="buddy">
        <cstring>txtLugarSalida</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="myQLineEdit" name="txtLugarSalida"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Fecha inicio</string>
       </property>
       <property name="buddy">
        <cstring>dtFechaInicio</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dtFechaInicio">
       <property name="displayFormat">
        <string>d/M/yyyy</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Fecha fin</string>
       </property>
       <property name="buddy">
        <cstring>dtFechaFin</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dtFechaFin">
       <property name="displayFormat">
        <string>d/M/yyyy</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Etapas</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QTreeView" name="twEtapas"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Etapa nº</string>
               </property>
               <property name="buddy">
                <cstring>spNumeroEtapa</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spNumeroEtapa">
               <property name="value">
                <number>1</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Salida</string>
               </property>
               <property name="buddy">
                <cstring>txtSalidaEtapa</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="myQLineEdit" name="txtSalidaEtapa"/>
             </item>
             <item>
              <widget class="QPushButton" name="btQuitarSalidaEtapa">
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Llegada</string>
               </property>
               <property name="buddy">
                <cstring>txtLlegadaEtapa</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="myQLineEdit" name="txtLlegadaEtapa"/>
             </item>
             <item>
              <widget class="QPushButton" name="btQuitarLlegadaEtapa">
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Fecha inicio</string>
               </property>
               <property name="buddy">
                <cstring>dtFechaInicioEtapa</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateEdit" name="dtFechaInicioEtapa">
               <property name="displayFormat">
                <string>d/M/yyyy</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Fecha fin</string>
               </property>
               <property name="buddy">
                <cstring>dtFechaFinEtapa</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateEdit" name="dtFechaFinEtapa">
               <property name="displayFormat">
                <string>d/M/yyyy</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="btAnadir">
               <property name="text">
                <string>Aña&amp;dir</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Notas</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="WidgetNotas" name="wdNotas" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btCancelar">
       <property name="text">
        <string>Can&amp;celar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btOK">
       <property name="text">
        <string>&amp;Aceptar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myQLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">widgets/myqlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>WidgetNotas</class>
   <extends>QWidget</extends>
   <header>widgets/widgetnotas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>spMisionNumero</tabstop>
  <tabstop>txtLugarSalida</tabstop>
  <tabstop>dtFechaInicio</tabstop>
  <tabstop>dtFechaFin</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>spNumeroEtapa</tabstop>
  <tabstop>txtSalidaEtapa</tabstop>
  <tabstop>btQuitarSalidaEtapa</tabstop>
  <tabstop>txtLlegadaEtapa</tabstop>
  <tabstop>btQuitarLlegadaEtapa</tabstop>
  <tabstop>dtFechaInicioEtapa</tabstop>
  <tabstop>dtFechaFinEtapa</tabstop>
  <tabstop>btAnadir</tabstop>
  <tabstop>btCancelar</tabstop>
  <tabstop>btOK</tabstop>
  <tabstop>twEtapas</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
